containerz/containerz.proto

```diff
@@ -149,6 +185,37 @@ message ImageTransferSuccess {
   uint64 image_size = 3;
 }
 
+message RemoveImageRequest {
+  // The container image name to be removed.
+  string name = 1;
+
+  // The container tag to be removed.
+  string tag = 2;
+
+  // If force is set, the target should attempt to forcibly remove the
+  // container.
+  bool force = 3;
+}
+
+message RemoveImageResponse {
+  enum Code {
+    // An unspecified error. Must use the detail value to describe the issue.
+    UNSPECIFIED = 0;
```

Comment:
From https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum -- we should generally avoid assigning semantics to the `_UNSPECIFIED`/0 value. Probably you want `INVALID` and `UNKNOWN` or similar here?

Reply:
Thanks - done.